config/routes.js

```diff
@@ -1,7 +1,9 @@
 const router = require('express').Router();
+const getServices = require('../app/middleware/getServices');
```

Comment:
It would be good to have the internal module imports alphabetised as there is quite a few already.

Reply:
I thought they were, bar the `external` one which is the router.